styleguide/source/assets/scss/03-organisms/_ipe-actions.scss

```diff
@@ -0,0 +1,151 @@
+/* Styling for the IPE Actions blocks when viewed in the AMA theme. */
+
+.contextual-region {
+  &:hover div.ipe-actions {
+    background: #f9f9f9;
+  }
+}
+
+// Specify the div selector to override default styling from panels-ipe.css.
+div.ipe-actions {
+  flex-flow: row wrap;
+  padding: 0 14px;
+  border-bottom: dotted 1px #cecece;
+
+  h5 {
+    flex-basis: 100%;
+    margin-bottom: 0;
+    padding: 8px 5px 0;
+    text-transform: none;
+  }
+
+  a {
+    color: $black-55;
+  }
+}
+
+.ipe-action-list {
+  .ipe-icon {
+    height: auto;
+    margin: 0;
+    font-size: 20px;
+  }
+}
+
+.ipe-action-list [data-action-id="move"] {
+  order: 1;
+
+  select {
```

Comment:
should this be ama_select or something along that vein instead of using an html element?

Reply:
We don't have the ability to easily modify the markup for this, so I used the elements as selectors here.